charts/cluster-autoscaler/values.yaml

```diff
@@ -366,6 +366,17 @@ rbac:
     name: ""
     # rbac.serviceAccount.automountServiceAccountToken -- Automount API credentials for a Service Account.
     automountServiceAccountToken: true
+  # rbac.additionalRule -- Additional rule for role/clusterrole
+  additionalRule: []
```

Comment:
Would `additionalRules` be more intuitive, since this value is an array?

Reply:
Yep `additionalRules` sounds better, I already made the change.

Reply:
nice!